variants/particle_xenon/variant.h

```diff
@@ -20,6 +20,7 @@
 #define _VARIANT_PARTICLE_XENON_
 
 #define _PINNUM(port, pin)    ((port)*32 + (pin))
+#define NRF52
```

Comment:
please remove this, you should either use NRF52_SERIES or NRF5232_XXAA or NRF52840_XXAA, which is std Nordic macros to detect nrf variants.
https://github.com/adafruit/Adafruit_nRF52_Arduino/blob/master/platform.txt#L60

Reply:
Sounds good, will do! I had put that in for testing and forgot to remove it.